src/Orc.CsvTextEditor/Converters/TextToTextArrayMultiValueConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="TextToTextArrayMultiValueConverter.cs" company="WildGums">
+//   Copyright (c) 2008 - 2016 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Orc.CsvTextEditor.Converters
+{
+    using System;
+    using System.Globalization;
+    using System.Windows.Data;
+
+    public class TextToTextArrayMultiValueConverter : IMultiValueConverter
+    {
+        #region Methods
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            return values.Clone();
+        }
+
+        public object[] ConvertBack(object value, Type[] targetTypes, object parameter, CultureInfo culture)
+        {
+            throw new NotImplementedException();
```

Comment:
hm... from my point of view- we can't reach this code... and when (probably) program will change to reach this code -  it wouldn't be silent... this exception wil tell us what to implement